src/helpers/pile-utilities.js

```diff
@@ -1769,11 +1769,11 @@ export async function rollTable({
 	if (game.modules.get("better-rolltables")?.active) {
 		results = (await game.betterTables.roll(table)).itemsData.map(result => {
 			return {
-				documentCollection: result.compendiumName || result.documentName,
-				documentId: result.item.id,
-				text: result.item.text || result.item.name,
-				img: result.item.img,
-				quantity: result.quantity
```

Comment:
`result.flags["better-rolltables"]["brt-result-formula"].formula` contains the formula. But it does not return the actual rolled quantity anymore instead there's that many results.
If I returned the formula of 3 in the quantity field I would get 9 instead of 3 Basic Poisons in the pile.